doc/docs/master/how-tos/working-with-data-and-pipelines.md

```diff
@@ -88,6 +102,26 @@ To build a Docker image, complete the following steps:
 If you decide to use DockerHub, follow the [Docker Hub Quickstart](https://docs.docker.com/docker-hub/) to
 create a repository for your project.
 1. Create a `Dockerfile` for your project. See the [OpenCV example](https://github.com/pachyderm/pachyderm/blob/master/examples/opencv/Dockerfile).
+1. Build a new image from the `Dockerfile` by specifying a tag:
+
+   ```bash
+   docker build -t <IMAGE>:<TAG> .
+   ```
+
+For more information about building Docker images, see
+[Docker documentation](https://docs.docker.com/engine/tutorials/dockerimages/).
+
+## Step 3: Push Your Docker Image to a Registry
+
+After building your image, you need to upload the image into
+a public or private image registry, such as
+[DockerHub](https://hub.docker.com) or other.
+
+Alternatively, you can use the Pachyderm's built-in functionality to
+tag, build, and push images by running the `pachctl update pipeline` command
+with the `--build` and `--push-images` flags. For more information, see
```

Comment:
```suggestion
with the `--build` or `--push-images` flag. For more information, see
```

It's redundant to pass both `--build` and `--push-images`, and it'll trigger a warning